modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLCopyEntryDisplayContext.java

```diff
@@ -54,22 +56,19 @@ public DLCopyEntryDisplayContext(
 		_httpServletRequest = httpServletRequest;
 		_liferayPortletResponse = liferayPortletResponse;
 		_themeDisplay = themeDisplay;
-	}
 
-	public String getActionURL() {
-		String actionName = "/document_library/copy_file_entry";
+		_copyDLObjectURL = null;
+	}
```

Comment:
Just a heads up. @jaimelr10 has made further changes here by Brian's request, so be aware of conflicts.

Reply:
thanks, when merged on master, I will rebase all the changes

<img src="https://media3.giphy.com/media/cFkiFMDg3iFoI/giphy.gif"/>